managed/devops/bin/cluster_health.py

```diff
@@ -210,21 +210,17 @@ class KubernetesDetails():
     def __init__(self, node_fqdn, config_map):
         self.namespace = node_fqdn.split('.')[2]
         self.pod_name = node_fqdn.split('.')[0]
```

Comment:
minor: I guess even this part will become problematic for multicluster. We need to ideally pass in some kind of a "node-name" -> (ns, pod name, kubeconfig) map in the future